views/layout.pug

```diff
@@ -0,0 +1,61 @@
+						symbol#faq(viewbox="0 0 24 24")
+							path(fill-rule="evenodd" d="M12 22C6.486 22 2 17.515 2 12 2 6.487 6.486 2 12 2s10 4.486 10 10c0 5.515-4.486 10-10 10zm0-18c4.411 0 8 3.589 8 8s-3.589 8-8 8-8-3.589-8-8 3.589-8 8-8zm1 11v-1.125A4.011 4.011 0 0 0 16 10c0-2.205-1.795-4-4-4s-4 1.795-4 4h2a2 2 0 1 1 2 2h-1v3h2zm-1 3.25a1.25 1.25 0 1 0 0-2.5 1.25 1.25 0 0 0 0 2.5z")
+						symbol#home(viewbox="0 0 24 24")
+							path(fill-rule="nonzero" d="M5.828 11l-2 2h-2l-.242-.586L3 11h2.828zM6 11v9h2v-7h8v7h2v-9l-6-6.172L6 11zm4 11H4v-9H1L12 2l11 11h-3v9h-6v-7h-4v7z")
+						symbol#github(viewbox="0 0 24 24")
+							path(d="M13.9 16.653c.4-.103.8-.103 1.199-.206 1.198-.31 2.197-.93 2.696-2.067.6-1.24.699-2.48.4-3.823-.1-.62-.4-1.033-.8-1.55-.1-.103-.1-.206-.1-.31.2-.826.2-1.549-.1-2.376 0-.103-.1-.206-.299-.206-.5 0-.899.206-1.298.413-.4.207-.7.413-.999.62-.1.103-.2.103-.3.103a9.039 9.039 0 0 0-4.693 0c-.1 0-.2 0-.3-.103-.698-.413-1.298-.827-2.096-.93-.5-.103-.5-.103-.6.413a4 4 0 0 0 0 2.17v.207c-.898 1.033-1.098 2.376-.898 3.616.1.413.1.723.2 1.136.499 1.447 1.497 2.273 2.995 2.687.4.103.8.206 1.298.31-.3.31-.499.826-.599 1.24 0 .103-.1.103-.1.103-.998.413-2.097.31-2.796-.827-.3-.516-.699-.93-1.398-1.033h-.5c-.199 0-.199.207-.099.31l.2.207c.499.31.898.826 1.098 1.446.4.93 1.099 1.343 2.097 1.447.4 0 .9 0 1.398-.104v1.963c0 .31-.3.517-.699.414a13.25 13.25 0 0 1-2.396-1.24c-2.996-2.17-4.594-5.166-4.394-8.989.2-4.753 3.595-8.575 8.089-9.505C15.298 1.156 20.29 4.462 21.69 9.73c1.298 5.166-1.598 10.538-6.392 12.192-.499.206-.799 0-.799-.62v-2.48c.1-.827 0-1.55-.599-2.17z")
+				ul.c-navbar-menu
+					li.c-navbar-menu-item
+						a.c-navbar-menu-logo(href="/")
+							svg.o-icon
```

Comment:
if we ever need to change the styling of the icon then you will need the component icon class as well for these...